src/Contacts/include/BipedalLocomotion/Contacts/GlobalCoPEvaluator.h

```diff
@@ -0,0 +1,114 @@
+/**
+ * @file GlobalCoPEvaluator.h
+ * @authors Giulio Romualdi
+ * @copyright 2023 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the BSD-3-Clause license.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_CONTACTS_GLOBAL_COP_EVALUATOR_H
+#define BIPEDAL_LOCOMOTION_CONTACTS_GLOBAL_COP_EVALUATOR_H
+
+#include <initializer_list>
+#include <vector>
+
+#include <Eigen/Dense>
+
+#include <BipedalLocomotion/Contacts/Contact.h>
+#include <BipedalLocomotion/System/Advanceable.h>
+
+namespace BipedalLocomotion
+{
+namespace Contacts
+{
+
+/**
+ * GlobalCoPEvaluator is a class that computes the global CoP given a set of contact wrenches.
+ * Following P. Sardain and G. Bessonnet, "Forces acting on a biped robot. Center of
+ * pressure-zero moment point," in IEEE Transactions on Systems, Man, and Cybernetics,
+ * we defined the global CoP as the weighted average of the CoP of each contact, with the weight
+ * determined by the normal force of the contact.
```

Comment:
Can you point to the formula used? I could not found the proposed formula in the paper.

Reply:
Hi @traversaro it should be the following equation. 

![image](https://github.com/ami-iit/bipedal-locomotion-framework/assets/16744101/4d8d8829-2043-457d-b4be-84d771cee72f)

Still I can add the equation in the documentation

Reply:
But that is not the " weighted average of the CoP of each contact", right? This is the COP of the sum of the contact forces, and it can be proven that the two definitions are the same, with the assumptions that all the contacts occur in the same plane. So (nitpick) I would write "Consistently with", not "Following".

Reply:
Here I wrote the math https://github.com/ami-iit/bipedal-locomotion-framework/pull/745/commits/881e92d4b9dd405b8f61b9dd9c888ca952a7f3b1